tests/conftest.py

```diff
@@ -123,7 +144,7 @@ def create_backend(backend_name: str, firmware: Firmware, display: bool, log_apd
 # If your tests needs to be run on independent Speculos instances (in case they affect
 # settings for example), then you should change this fixture scope and choose between
 # function, class, module or session.
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
why do you need this?

Reply:
The tests failed without this :(
I suggest we merge as-is and investigate after

Reply:
Only fails on Stax right?

Reply:
Yes